SYS_ATL/API.py

```diff
@@ -1031,6 +1031,22 @@ def par_to_seq(self, par_pattern):
 
         return Procedure(loopir, _provenance_eq_Procedure=self)
 
+    def lift_alloc_simple(self, alloc_site_pattern, n_lifts=1):
```

Comment:
I don't understand why this is a totally different API from `lift_alloc` rather than an option.

Reply:
I'm planning to eventually implement `lift_alloc` using `lift_alloc_simple`, `expand_dim`, and `rearrange_dim` . I think we talked about this in a meeting in December

Reply:
Okay, we can discuss naming later, then. It seems confusing to me to have both `lift_alloc` and `lift_alloc_simple` in the same system.

Reply:
I think our plan is to eventually replace `lift_alloc` with `lift_alloc_simple`. I think it's fine for both to coexist for a while before we rewrite all the tests

Reply:
Sounds good to me!